src/classes/CON_CannotDelete_TDTM.cls

```diff
@@ -48,19 +48,21 @@ public with sharing class CON_CannotDelete_TDTM extends TDTM_Runnable {
     ********************************************************************************************************/
     public override DmlWrapper run(List<SObject> newlist, List<SObject> oldlist, 
     TDTM_Runnable.Action triggerAction, Schema.DescribeSObjectResult objResult) {
-
+      
```

Comment:
Can we update the method description and class description to include the new objects we're now checking?

Reply:
Thanks!